### ISSUE TITLE
Had you ever use other backbone?

### ISSUE THREAD

**Issue by ElvishElvis** [supernotman/RetinaFace_Pytorch#13]
Thanks for your great job!
I'd use mobilenet V1 0.25 to replace your resnet ,however, I found it really hard to converge.
Although the loss was quite low even at the first several epochs, but it just keep that way forever.
Had you tried other light-weight backbone for your code? Could you share some details for your training?
Also, I am trying to increase # landmarks to 68 with the 300w dataset with your code, had you ever tried that?
Thanks!

**Comment by supernotman** [comment]
> Thanks for your great job!
> I'd use mobilenet V1 0.25 to replace your resnet ,however, I found it really hard to converge.
> Although the loss was quite low even at the first several epochs, but it just keep that way forever.
> Had you tried other light-weight backbone for your code? Could you share some details for your training?
> Also, I am trying to increase # landmarks to 68 with the 300w dataset with your code, had you ever tried that?
> Thanks!

Only resnet50 used.But you can use resnet101 or resnet152 simply with：
retinaface = torchvision_model.create_retinaface(return_layers，backbone_name='resnet101')
I will test mobilenet v1 in soon.

**Comment by ElvishElvis** [comment]
with MobileNet 0.25, and use CelebA dataset, I can get Recall 0.92 Precision 0.89 at the first epoch

**Comment by supernotman** [comment]
> with MobileNet 0.25, and use CelebA dataset, I can get Recall 0.92 Precision 0.89 at the first epoch

By modify this code?Or in your own way.

**Comment by ElvishElvis** [comment]
with mainly your code, but with many modification.

Here's the current data:
100%|████████████████████████████████████████████████████████████████████████████████| 375/375 [01:29<00:00,  2.40it/s]
Recall: 0.9753333333333334
Precision: 0.965111111111111
landmark:  2.2613525451660155
miss: 0
Current lr is 0.00039021130325903074

**Comment by ElvishElvis** [comment]
landmark is the MSE loss for landmarks

**Comment by ElvishElvis** [comment]
Can you add my wechat: 872996860 and maybe we can share more? I am currently finetuning my model for 68 points landmark with RetinaFace.

**Comment by wangpupanjing** [comment]
@ElvishElvis 
hello    can you provide your code  with mobilenet0.25?.....3Q

**Comment by ElvishElvis** [comment]
> @ElvishElvis
> hello can you provide your code with mobilenet0.25?.....3Q

hello please refer to my github, where you can find my older version of the code for reference.
